src/containers/ToDoList/ToDoList.jsx

```diff
@@ -81,22 +102,54 @@ export const ToDoList = () => {
         ))}
 
         <StyledWrapperSelect>
-          <StyledParagraph>Items left</StyledParagraph>
+          <StyledParagraph>
+            {tasks.filter((task) => task.checked).length} items left
+          </StyledParagraph>
           <StyledWrapperSelectButtonGroup>
-            <button>All</button>
-            <button>Active</button>
-            <button>Completed</button>
+            <StyledFilterButton
+              isActive={filterTask === "all"}
```

Comment:
Instead of using just `string`s here, please create an enum object, that will represent the states of filters.